percent_encoding/src/lib.rs

```diff
@@ -101,14 +101,26 @@ impl AsciiSet {
         AsciiSet { mask }
     }
 
+    pub const fn add_range(&self, start: u8, end: u8) -> Self {
```

Comment:
If this took [Range](https://doc.rust-lang.org/std/ops/struct.Range.html) instead of two vars, then it would be more obvious that this was the half-open range (it's actually the inclusive range, but that's not obvious from the method name at all)